README.rst

```diff
@@ -1,10 +1,17 @@
+
+#########
+VCR.py 📼
+#########
+
+
 |PyPI| |Python versions| |Build Status| |CodeCov| |Gitter| |CodeStyleBlack|
 
-VCR.py
-======
+----
+
+.. image:: https://raw.github.com/kevin1024/vcrpy/v4.0.x/vcr.svg?sanitize=true
```

Comment:
This line will have to change the branch section of the path to master before publishing.

I set it this way so I can preview the readme and the sphinx build of the docs.

https://github.com/kevin1024/vcrpy/tree/v4.0.x